ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiCommandService.scala

```diff
@@ -91,7 +90,7 @@ private[apiserver] final class ApiCommandService private[services] (
         submissionTracker.track(CommandSubmission(commands, timeout))
       } else {
         Future.failed(
-          new ApiException(Status.UNAVAILABLE.withDescription("Service has been shut down."))
+          ErrorFactories.serviceNotRunning(definiteAnswer = Some(false))
```

Comment:
Would we ever return this with definitive set to true?

Reply:
I can imagine that for certain implementations, a service may be turned off and it's not even configurable.